pkg/services/tree/service.go

```diff
@@ -522,6 +522,51 @@ func (s *Service) GetOpLog(req *GetOpLogRequest, srv TreeService_GetOpLogServer)
 	}
 }
 
+func (s *Service) GetTrees(ctx context.Context, req *GetTreesRequest) (*GetTreesResponse, error) {
```

Comment:
I don't like `Get` here, we have `GetOpLog` (which _is_ a "gettree" operation) and `GetAddByPath`. What about `List`?

Reply:
renamed: `ListTrees` in RPC and `TreeList` in the `Forest` intreface